extensions/spotify-controls/src/play.ts

```diff
@@ -1,5 +1,23 @@
 import { runAppleScriptSilently } from "./utils";
 
 export default async () => {
-  await runAppleScriptSilently('tell application "Spotify" to play');
+  const script = `
+    tell application "Spotify"
+
+      if not application "Spotify" is running then
+        
+        activate
+
+        repeat until application "Spotify" is running
+          delay 1
+        end repeat
+
+      end if
+
+      play
+        
+    end tell
```

Comment:
```suggestion
  const script = `
    tell application "Spotify"
      if not application "Spotify" is running then
        activate
        repeat until application "Spotify" is running
          delay 1
        end repeat
      end if
      play
    end tell
```

Reply:
Good suggestion.  This commit is outdated.